nixos/tests/coturn.nix

```diff
@@ -0,0 +1,29 @@
+import ./make-test-python.nix ({ ... }: {
+  name = "coturn";
```

Comment:
Can you add `passthru.tests.coturn = nixosTests.coturn;` to the coturn package?

Reply:
Also this test should be linked in `nixos/tests/all-tests.nix`.

Reply:
Is 6be0a41 correct?